sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample1_RecognizeReceipts.md

```diff
@@ -0,0 +1,74 @@
+# Recognize receipts
+
+This sample demonstrates how to recognize and extract common fields from US receipts, using a pre-trained receipt model.
+
+To get started you'll need a Cognitive Services resource or a Form Recognizer resource.  See [README][README] for prerequisites and instructions.
```

Comment:
Can you link more specifically to the section where you create the resource in the readme? Is it also possible to link to how to get the endpoint and apiKey? (Doesn't have to be per sample, you can also include this info in the samples readme since every sample is going to need it)

Reply:
fixed